src/tree/GitHubConfigGroupTreeItem.ts

```diff
@@ -49,16 +76,20 @@ export class GitHubConfigGroupTreeItem extends AzExtParentTreeItem {
 
             for (const yamlFile of yamlFiles) {
                 const yamlFilePath: string = join(workflowsDir, yamlFile);
-                const buildConfigs: BuildConfigs | undefined = await parseYamlFile(context, yamlFilePath);
-                buildConfigs && treeItems.push(new GitHubConfigGroupTreeItem(parent, yamlFilePath, buildConfigs));
+                try {
```

Comment:
It's kind of a small distinction, but could you call `refreshImpl` here to initialize the tree item?  Right now we're doing two similar things in two places.